### ISSUE TITLE
The order form generates tasks with missing information

### ISSUE THREAD

**Issue by rayon9coop** [coopcycle/coopcycle-web#3663]
**Describe the bug
1. The order form asks for the pick-up and drop-off address but does not ask for the names and telephone numbers of the people where the courier will pick up and drop off the parcel, even though this information is necessary for couriers. 
2. The order form asks for the name and telephone number of the contact person (the person executing the delivery order), but only copies this person's name and not their telephone number in the tasks.

**To reproduce the behaviour
Steps to reproduce the behaviour :
1. Go to https://www.rayon9.be/commander-une-livraison/
2. Fill in all the fields 
3. Confirm
4. Check the tasks created on the dashboard: there is no telephone number or name for the pick-up and drop-off (bug 1).
5. Check the tasks created on the dashboard: there is no telephone number for the contact person (bug 2)

**Expected behaviour**
Have all the information for the couriers

**Comment by rayon9coop** [comment]
I could but I think it's a pity, it seems to me that my description is super clear, no ;-)?

**Comment by rayon9coop** [comment]
More seriously, i explained this to Antoine at the last GA. He understood and noted. The general idea is the same than this: https://github.com/coopcycle/coopcycle-web/issues/2418#issuecomment-1772330248

Asking for the same info whereever the tasks are created

**Comment by Paul-Eraman-CoopCycle** [comment]
i think i understand actually, yes 

how many of these types of orders do you get from the embed?

**Comment by rayon9coop** [comment]
Not so much. 
Behind your question, I understand that it won't be a priority, which I can understand as a valid argument in regard of the ressources of coopcycle's dev team vs fires to put out. Just notice that requests that can be encoded by individuals on the form integrated on our site save a lot of time. Without this solution, the cost of executing this type of request eats up all the benefit. So please don't close this issue ;-)

**Comment by Paul-Eraman-CoopCycle** [comment]
Yep exactly. Not closing, but keeping it low on the list of priorities for the moment until we have things a bit more under control
